modules/rest-api/pages/load-trusted-cas.adoc

```diff
@@ -15,9 +15,11 @@ POST /node/controller/loadTrustedCAs
 [#description]
 == Description
 
-Loads trusted CA (or _root_) certificates that reside on the accessed node into the Couchbase-Server cluster.
-All loaded CA certificates can be accessed by all nodes.
-Loaded certificates can be used to provide authority to the cluster's nodes, and can be used to authenticate clients' access-attempts.
+Loads trusted certificates that reside on the accessed node into the Couchbase-Server cluster.
+All loaded certificates can be accessed by all nodes.
+Loaded CA (or _root_) certificates can be used to provide authority to the cluster's nodes, and can be used to authenticate clients' access-attempts.
+Loaded _intermediate_ certificates are provided to clients when secure client-access is attempted, so that the node's chain of authority can be inspected.
```

Comment:
> Loaded _intermediate_ certificates are provided to clients when secure client-access is attempted, so that the node's chain of authority can be inspected.

That's not the goal, but rather a side effect here. We should not mention it.

Reply:
I've removed.